setup.py

```diff
@@ -66,69 +73,62 @@ def main():
                             help="Pass arguments to ninja. Usage: --ninja=\"args...\". Defaults to \"-j <cpu count>\"")
     args = parser.parse_args()
 
+    # ROS2 standalone obviously implies ROS2
+    if args.with_ros2_standalone:
+        args.with_ros2 = True
+
+    # Go to script directory
+    os.chdir(sys.path[0])
+
+    # Prepare build directory
+    if args.clean_build and os.path.isdir(args.build_dir):
+        shutil.rmtree(args.build_dir, ignore_errors=True)
```

Comment:
`radar_msgs` could be removed here, but wasn't sure if that's the intended behavior - would be inconsistent with not removing vcpkg-originating packages.

Reply:
IMO, it wouldn't be an intended behavior. Dependencies are installed once per machine and there is no need to reinstall them (unless there is a version update - rare case). `--clean-build` as described in `argparse` only removes the build directory before cmake.